src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -436,6 +437,44 @@ object Helpers extends SparkSessionWrapper {
     spark.conf.set("spark.databricks.delta.vacuum.parallelDelete.enabled", "false")
     s"SHRED COMPLETE: ${target.tableFullName}"
   }
+  /**
+   * Execute a parallelized streaming to follow the instructions provided through CloneDetail class
+   *
+   * @param cloneDetails details required to execute the parallelized clone
+   * @return
+   */
+  def tableStream(cloneDetails: Seq[CloneDetail]): Seq[CloneReport] = {
```

Comment:
I think we need to persist the clone report. This brings more credence to the idea of refactoring the snapshots out of these helper sections to allow for a more complex / controlled workflow to happen. I even think there could be a main class for snapshotting since everyone will be doing this from a scheduled job anyway.